apps/src/StudioApp.js

```diff
@@ -98,15 +98,26 @@ var codegen = require('./lib/tools/jsinterpreter/codegen');
 var copyrightStrings;
 
 /**
- * The minimum width of a playable whole blockly game.
+ * Store experiment parameters.
  */
+const isBigPlayspaceExperiment = experiments.isEnabledAllowingQueryString(
+  experiments.BIG_PLAYSPACE
+);
+const bigPlaySpacePadding = queryParams('bigPlayspacePadding') || 160;
+
+/**
+ * Get the maximum resizable width of the playspace.
+ */
+const getMaxResizableVisualizationWidth = () => {
+  return isBigPlayspaceExperiment
+    ? Math.min(window.innerHeight - bigPlaySpacePadding, window.innerWidth / 2)
+    : 400;
+};
+
 const MIN_WIDTH = 1400;
 const DEFAULT_MOBILE_NO_PADDING_SHARE_WIDTH = 400;
-export const MAX_VISUALIZATION_WIDTH = experiments.isEnabled(
-  experiments.BIG_PLAYSPACE
-)
-  ? 0.75 * window.innerHeight
-  : 400;
+const DEFAULT_VISUALIZATION_WIDTH = 400;
+export const MAX_VISUALIZATION_WIDTH = 400;
```

Comment:
Is this still used somewhere? I don't see where it's imported in this PR.

Reply:
Yes, Dance Party still uses it here.  
https://github.com/code-dot-org/code-dot-org/blob/3d4b6c1e827fd8a3a46c2e3d237ada73f2c5d5ef/apps/src/dance/constants.ts#L1-L7